test/fixtures/standalone/variables.tf

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "data_ingestion_project_id" {
```

Comment:
Where do we use these variables?

Reply:
It is being used on [test/integration/standalone/standalone_test.go](https://github.com/renato-rudnicki/terraform-google-secured-data-warehouse/blob/b1d56764cce23dd51441c5d8980e04a97d420e6f/test/integration/standalone/standalone_test.go#L48)